.github/workflows/ci.yml

```diff
@@ -33,3 +33,38 @@ jobs:
       - run: nix build -L .#charon
       - run: nix build -L .#charon-ml
       - run: nix flake check -L
+
+  eurydice:
+    needs: [check_if_skip_duplicate_job, nix]
+    if: needs.check_if_skip_duplicate_job.outputs.should_skip != 'true'
+    runs-on: [self-hosted, linux, nix]
+    steps:
+      - run: nix build -L github:aeneasverif/eurydice --override-input charon github:aeneasverif/charon/${{ github.ref }}
+
+  kyber:
+    needs: [check_if_skip_duplicate_job, eurydice]
+    if: needs.check_if_skip_duplicate_job.outputs.should_skip != 'true'
+    runs-on: [self-hosted, linux, nix]
+    steps:
+      - name: checkout libcrux
+        uses: actions/checkout@v4
+        with:
+          repository: cryspen/libcrux
+      - name: checkout eurydice
+        uses: actions/checkout@v4
+        with:
+          repository: aeneasverif/eurydice
+          path: eurydice
```

Comment:
Could you add a comment explaining that `kyber` expects a checkout of eurydice and not just the binary?

Reply:
done!